SPECS/dbus/dbus.spec

```diff
@@ -16,6 +16,8 @@ BuildRequires:  expat-devel
 BuildRequires:  libselinux-devel
 BuildRequires:  systemd-bootstrap-devel
 BuildRequires:  xz-devel
+BuildRequires:  filesystem
+Requires:       filesystem
```

Comment:
The additional dependencies are probably not wrong - I'm guessing everything needs `filesystem` - but why the additional entries?

Reply:
I had added these as it came up during a test iso build that I was doing locally (this may have been too specific of a scenario to matter). But since the `filesystem` package specifically sets up the "messagebus" group and it is used in the %attr added to the %files section it was required to actually have the chmod run successfully.